app/src/main/java/com/example/movieapp/crawler/pojo/AllData.kt

```diff
@@ -0,0 +1,6 @@
+package com.example.movieapp.crawler.pojo
+
+/**
+ * Created by Benjamin Vouillon on 08,July,2020
+ */
+data class AllData(val page:Int, val total_results:Int, val total_pages:Int, val results: List<Movie>)
```

Comment:
Use a more explicit name for this class. It's not all the data. It's a single page **result** of a request of **movies** using a specific filter. 

There's again some snake case variables here ;-)

Reply:
changed.